### PR TITLE
chore(ui): Update metadata header styling

### PR DESCRIPTION
**Pull request by m-rgba**
## Description

| Before    | After |
| -------- | ------- |
| before | after

- Updates metadata header styling on ops and objects.

### CODE REVIEW

**Comment by m-rgba** [comment]
Updated FeatureBee link: https://beta.wandb.ai/?betaVersion=9aca7b7ec29959c485e3809c806b95324672bfd7

**Comment by m-rgba** [comment]
Updated FeatureBee link: https://beta.wandb.ai/?betaVersion=57e58d13a1459a4f170968bedeac8059b3df84ec

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ObjectVersionPage.tsx [hunk 1]
```diff
@@ -211,47 +211,50 @@ const ObjectVersionPageInner: React.FC<{
...
-                      filter={{
-                        objectName,
-                      }}
-                      versionCount={objectVersionCount}
-                      neverPeek
-                      variant="secondary"
+        <Tailwind>
+          <div className="grid w-full auto-cols-max grid-flow-col gap-[16px] text-[14px]">
+            <div className="block">
+              <p className="text-moon-500">Name:</p>
```
Comment (jamie-rasmussen) [inline weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ObjectVersionPage.tsx:217]: Can be design's call but I think these headers might look better without the trailing `:`

**Review by jamie-rasmussen — APPROVED** [review]

File: weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ObjectVersionPage.tsx (hunk 1 above)
Comment (m-rgba) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ObjectVersionPage.tsx:217]: Let's do it, will update

**Pull request merged**